packages/react-storage/src/components/StorageBrowser/views/LocationActionView/index.ts

```diff
@@ -27,7 +27,6 @@ export {
   UploadViewState,
   useUploadView,
 } from './UploadView';
-export { useActionView } from './useActionView';
```

Comment:
I lost the track for this one. Is this intended?

Reply:
Yes, it was an unused "base" hook that was to be used for dynamically generating `useView` hooks. Came across while fixing imports and removed it